Formula/vlang.rb

```diff
@@ -0,0 +1,21 @@
+class Vlang < Formula
+  desc "V programming language"
+  homepage "https://vlang.io"
+  url "https://github.com/vlang/v/archive/0.1.20.tar.gz"
+  sha256 "8102b48b2c82be6be14633e76e71e215aab5221198315436f97be53e1abe1f5d"
+
+  def install
+    system "make"
+    bin.install "v"
+    bin.install "vlib","compiler","examples","thirdparty","tools"
```

Comment:
This doesn't seem right. This will install non-binaries to the `bin` folder which is not allowed. These should go in appropriate folders like `lib` or `share`.

If this software requires everything to be relative to the `v` binary, everything should be installed to `libexec` and a symlink to `v` be placed in `bin`.

An issue should be filed upstream to add an actual `make install` statement.

Reply:
now libexec is used.